modules/ROOT/pages/configure-monitoring-cloudhub.adoc

```diff
@@ -3,58 +3,22 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To start monitoring Mule apps that are managed in CloudHub, you must deploy them to a supported version of Mule runtime engine and set the runtime version in your Mule app. 
+To start monitoring Mule apps that are managed in CloudHub, you must deploy them to a supported version of Mule runtime engine and set the runtime version in your Mule app.
 
 == Supported Versions of Mule Deployed to CloudHub
 
-Anypoint Monitoring non-AM images receive patch updates as soon as they are available, while -AM images receive the latest patch updates only once per month. It is recommended that you switch to a non-AM image if your production environment requires a Mule runtime engine patch.
+Anypoint Monitoring non-AM images receive patch updates as soon as they are available. Anypoint Monitoring -AM images are no longer receiving patch updates. It is recommended that you switch to a non-AM image if your production environment requires a Mule runtime engine patch.
```

Comment:
```suggestion
Anypoint Monitoring non-AM images receive patch updates as soon as they are available. Because Anypoint Monitoring -AM images are no longer receiving patch updates, switch to a non-AM image if your production environment requires a Mule runtime engine patch.
```

Reply:
Doublechecking: the images are of type "non-AM" and "-AM" (with the hyphen)?
Does AM represent Anypoint Monitoring? Could we call the images simply "Anypoint Monitoring images" and "non-Anypoint Monitoring images"?

Reply:
```suggestion
Anypoint Monitoring with Mule Runtime images receive patch updates as soon as they are available. Because Anypoint Monitoring -AM images are no longer receiving patch updates, switch to a Mule Runtime (non-AM) image if your production environment requires a Mule runtime engine patch.
```